### PR TITLE
strip: Add stripFormatting option and by default do not strip <b>, <i>, or <u>

### PR DESCRIPTION
**Pull request by sparklingW**
#247

### CODE REVIEW

**Comment by sparklingW** [comment]
Could you please tell me where is formatting in Breadcrumbs component?

**Comment by raineorshine** [comment]
> Could you please tell me where is formatting in Breadcrumbs component?

Sure, it is here:
https://github.com/cybersemics/em/blob/dev/src/components/Breadcrumbs.js#L61

Although it looks like the actual formatting will take place in the Link component:
https://github.com/cybersemics/em/blob/dev/src/components/Link.js

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (raineorshine) [inline .idea/.gitignore]: Make sure to remove `.idea` from git. Adding an entry to the `.gitignore` will not remove the files from git if they have already been committed.

File: src/util/strip.js [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    .replace(/<(?:.|\n)*?>/gm, '')
+    .replace(/&nbsp;/gm, ' ')
+    .trim()
+  }
+  else {
+    return html
+      .replace(/<(?:.|\n)*?([^bui])>/gm, '')
+      .replace(/&nbsp;/gm, ' ')
+      .trim()
+  }
```
Comment (raineorshine) [inline src/util/strip.js]: Notice the redundancy between these two sections of code. Can you think of a way to reuse the common portions, while using a ternary operator to switch on just the part that is different?

**Pull request closed without merging**